ibm-build-lab/VAD-VAR-Workshop
# feat: Add model selector to tokenization demo

Relates to #226 

Content changes:

- New locale texts added
- Minor copy change preceding demo

Code changes:

- Add the capability to change models in the demo
- Ability to see internal representation as well 

Testing:
1. Start dev server and navigate to http://localhost:8000/en/watsonx/watsonxai/ref100
2. Ensure toggle and dropdown function as expected
